templates/cbs-tools/scripts/create_sig.sh

```diff
@@ -116,9 +116,9 @@ do
         ;;
         7) echo "* Checking distribution el$DIST configuration...";  ( ! $optiona ) && ARCHES="x86_64"; DEFAULT_DISTTAG="el7.centos"
         ;;
-        8) echo "* Checking distribution el$DIST configuration...";  ( ! $optiona ) && ARCHES="x86_64 aarch64 ppc64le"; DEFAULT_DISTTAG="el8"
+        8) echo "* Checking distribution el$DIST configuration...";  ( ! $optiona ) && ARCHES="x86_64 aarch64 ppc64le"; DEFAULT_DISTTAG="el8"; BUILDROOT_PKGS_EXTRAS="centpkg-minimal"
```

Comment:
Good to add centpkg-minimal for c8/c8s but why not for c7 too ? it was built so worth adding ? we still have some requests for c7 tags , so worth adding in case they want to build from git.centos.org ?